<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BUNMG7T" failures="1" tests="4" name="Test" time="0.469" errors="0" timestamp="2023-10-13T22:47:58 IST">
  <testcase classname="test.Demo" name="Demo1" time="0.122">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Demo.Demo1(Demo.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Demo1 -->
  <testcase classname="test.Demo" name="SecondTest" time="0.013"/>
  <testcase classname="test.Demo" name="ThirdTest" time="0.015"/>
  <testcase classname="test.Demo2" name="Ploan" time="0.027"/>
</testsuite> <!-- Test -->
